src/sentry/static/sentry/app/components/avatar/suggestedAvatarStack.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {css} from '@emotion/core';
+import styled from '@emotion/styled';
+
+import ActorAvatar from 'app/components/avatar/actorAvatar';
+import BaseAvatar from 'app/components/avatar/baseAvatar';
+import {Actor} from 'app/types';
+
+type Props = {
+  owners: Actor[];
+} & BaseAvatar['props'] &
+  Omit<ActorAvatar['props'], 'actor' | 'hasTooltip'>;
+
+// Constrain the number of visible suggestions
+const MAX_SUGGESTIONS = 5;
+
+const SuggestedAvatarStack = ({owners, ...props}: Props) => {
+  const backgroundAvatarProps = {
+    ...props,
+    round: owners[0].type === 'user',
+    suggested: true,
+  };
+  const numAvatars = Math.min(owners.length, MAX_SUGGESTIONS);
+  const backgroundAvatars: React.ReactElement[] = [];
+  // Generate N - 1 background avatars
+  for (let i = 0; i < numAvatars - 1; i++) {
+    backgroundAvatars.push(
```

Comment:
So this is just stylistic, but you could use a map here - JS doesn't have a range function like in python, but you can get something close with:

```
[...Array(numAvatars)].map((_, i) => i)
```

You need to spread or use `Array.from()`, otherwise it's an empty array of size `n` that you can't `map()`.